cli/mfg.c

```diff
@@ -44,6 +44,14 @@
 #include <stdio.h>
 #include <string.h>
 
+static const struct argconfig_choice secure_state_choices[] = {
+	{"INITIALIZED_UNSECURED", SWITCHTEC_INITIALIZED_UNSECURED,
+	 	"Unsecured"},
+	{"INITIALIZED_SECURED", SWITCHTEC_INITIALIZED_SECURED,
```

Comment:
These are confusing names... Why is everything INITIALIZED?

Reply:
@lsgunth The chip now has three state regarding security, UNINITIALIZED_UNSECURED, INITIALIZED_UNSECURED AND INITIALIZED_SECURED. With default shipping F/W, the chip is in the UNINITIALIZED_UNSECURED state, in which it can be changed into INITIALIZED_UNSECURED or INITIALIZED_SECURED state per user's need of the security features.

Once a chip is changed into a initialzed state, INITIALIZED_UNSECURED or INITIALIZED_SECURED, its state cannot be updated any more.

Reply:
Hmm, those don't sound like very intuitive names. This all needs to be documented better.